Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -530,7 +549,7 @@ public void indexConcepts(String userName)
                 String searchString = fieldMap.get(search.fieldName());
                 if (searchString != null) {
                     searchString = searchString.toLowerCase();
-                    buildQuery(occur, perFieldAnalyzerWrapper, qBuild, builder, luceneFieldAnnotation, searchString);
+                    buildQuery(occur, perFieldAnalyzerWrapper, qBuild, query, builder, luceneFieldAnnotation, searchString);
```

Comment:
what is 'query' used for? why is it a class variable? this is a multi-threaded environment, having a class variable that is changed in a method is a really bad idea and will cause all kinds of issues.

Reply:
removed the query variable